content/release/ReleaseNotes371.md

```diff
@@ -0,0 +1,88 @@
+
+#### Bug fixes:
+* Server-side React page rendering is instrumented by default.
+* Agent not functioning correctly with the latest Node.js version 12.16.0.
+* Use of `_headers` in the response object is deprecated. This was revised to use  `getHeaders()`.
+
+#### New features and improvements:
+This month's work contains internal architecture improvements, improved testing and test cases. Work is also progressing on new Protect functionality for the agent.The Node.js agent now supports the LoopBack 3 framework. Node.js language version 8 is no longer supported as of agent version 2.11.0.
+
+### Python Agent Summary 
+
+**Agent versions released this month:** 2.6.1, 2.7.0
+**Language versions supported:** 2.7, 3.5-3.7
+
+#### Bug fixes:
+* Python Assess is prevented from collecting findings with older versions of Werkzeug.
+* Using the Jinja2 templating engine causes false positives.
+* Compilation macro error occurs regarding reproducible builds on certain systems.
+* Django version info for older versions of the framework causes issues.
+
+#### New features and improvements:
+This month's work continued hardening Python Assess and included updates to vulnerability accuracy and reporting. The 2.7.0 version of the agent includes improved support for older versions of Django and Werkzeug, better stability for the pymysql, psycopg2, and pycassa database adapters, and greater specificity for XSS when using the Jinja templating engine.
+
+### Ruby Agent Summary 
+
+**Agent versions released this month:** 3.5.1, 3.5.2, 3.5.3, 3.5.4, 3.6.0 (minor)
+**Language versions supported:** 2.4-2.6
+
+#### Bug fixes:
+* Rails style parameters in which Hash keys are passed in as nested parameter names is not handled well.
+
+#### New features and improvements:
+This most recent minor release contains internal improvements, including a reduction in object creation resulting from monkey patching for security analysis. In addition, we have refactored our evaluation of constants to reduce startup time and enhanced our support for the prepend monkeypatch style favored by Rails 6.
+
+### .NET Agent Summary
+
+**Agent versions released this month:**
+NET Framework: 20.2.1, 20.2.2, 20.2.3 
+.NET Core: 1.2.0, 1.2.1, 1.2.2
+
+**Language versions supported:**
+.NET Framework: 3.5, 4.0, 4.5, 4.5.1, 4.5.2, 4.6, 4.6.1, 4.6.2, 4.7, 4.7.1, 4.7.2, 4.8
+.NET Core: 2.1, 2.2, 3.0, 3.1
+
+#### Bug fixes:
+* Observed routes were not reported if the .NET agent was configured to use a proxy for communication with Contrast.
+* .NET agent could stop using the session ID specified via configuration after a period of time.
+* The `x-www-form-urlencoded` parameters could be reported as parameter keys.
+* .NET Core agent would not change Protect rule modes after initialization.
+
+#### New features and improvements:
+Added new gadgets for Protect Untrusted Deserialization and a session timeout rule for .NET Core. Most significantly, this release adds support for .NET Core 3.0 and 3.1.
```

Comment:
We should find a way to raise the prominence for adding support for .NET core 3.0 and 3.1. This is definitely the most important part of the .NET Core agent release.

Reply:
Resolved.